ui/raidboss/data/03-hw/dungeon/the_vault.txt

```diff
@@ -0,0 +1,215 @@
+1125.1 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1130.3 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1134.4 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1139.4 "Faith Unmoving" sync /:Ser Grinnaux:1027:/
+
+1144.1 "Dimensional Collapse" sync /:Ser Grinnaux:1028:/
+1152.3 "Dimensional Rip" sync /:Ser Grinnaux:102C:/
+1155.5 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1161.6 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1166.8 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1170.8 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1177.4 "Dimensional Collapse" sync /:Ser Grinnaux:1028:/
+1182.5 "Faith Unmoving" sync /:Ser Grinnaux:1027:/
+
+1192.7 "Dimensional Rip" sync /:Ser Grinnaux:102C:/ window 15,15
+1195.9 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1202.1 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1207.9 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1212.1 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1218.7 "Dimensional Collapse" sync /:Ser Grinnaux:1028:/
+1223.8 "Faith Unmoving" sync /:Ser Grinnaux:1027:/
+
+1234.0 "Dimensional Rip" sync /:Ser Grinnaux:102C:/
+1237.1 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1243.3 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1248.8 "Hyperdimensional Slash" sync /:Ser Grinnaux:1026:/
+1253.0 "Heavy Swing" sync /:Ser Grinnaux:1025:/
+1259.6 "Dimensional Collapse" sync /:Ser Grinnaux:1028:/
+1264.7 "Faith Unmoving" sync /:Ser Grinnaux:1027:/ jump 1182.5
+
+# Closing sequence
+1441.9 "--sync--" sync /:Ser Grinnaux the Bull:10A0:/ window 500,10
+1442.0 "Retreating" duration 8
+1450.0 "Retreat" sync /:Ser Grinnaux the Bull:10A1:/
+
+
+###Ser Charibert
+
+2000.0 "--sync--" sync /The Chancel will be sealed off/ window 2000,0
+2006.3 "Altar Candle" sync /:Ser Charibert:1030:/
+2015.0 "Heavensflame" sync /:Ser Charibert:1031:/
+2019.2 "Holy Chain" sync /:Ser Charibert:1033:/
+2024.4 "Altar Candle" sync /:Ser Charibert:1030:/
+2028.7 "Knights Appear"
+2037.6 "Altar Pyre" sync /:Ser Charibert:1035:/
+
+2046.6 "Altar Candle" sync /:Ser Charibert:1030:/
+2055.3 "Heavensflame" sync /:Ser Charibert:1031:/
+2059.5 "Holy Chain" sync /:Ser Charibert:1033:/
+2064.7 "Altar Candle" sync /:Ser Charibert:1030:/
+2068.8 "Knights Appear"
+2077.9 "Altar Pyre" sync /:Ser Charibert:1035:/ window 15,15 jump 2037.6
+
+2086.9 "Altar Candle"
+2095.6 "Heavensflame"
+2099.8 "Holy Chain"
+2105.0 "Altar Candle"
+
+# Intermission at < 60% HP
+
+2200.0 "Unknown_1019" sync /:Ser Charibert:1019:/ window 200,5
+2205.1 "--sync--" sync /:Ser Charibert:1018:/
+2207.2 "--sync--" sync /:Ser Charibert:1036:/
+2218.4 "Black Knight's Tour" sync /:Dusk Knight:1039:/
+2218.5 "White Knight's Tour" sync /:Dawn Knight:1038:/
+2231.7 "Black Knight's Tour" sync /:Dusk Knight:1039:/
+2231.9 "White Knight's Tour" sync /:Dawn Knight:1038:/
+2245.0 "Black Knight's Tour" sync /:Dusk Knight:1039:/
+2245.2 "White Knight's Tour" sync /:Dawn Knight:1038:/
+2250 "--sync--" sync /:Ser Charibert:1037:/ window 50,1 jump 2294.0
```

Comment:
Does 1037 get used elsewhere? if no, I think this would be clearer to have `2294.0 "--sync--" sync /:Ser Charibert:1037:/ window 2300,0` and not this 2250 line.

Reply:
1037 is used whether or not the intermission is completed successfully. I suppose it's not terribly important to handle a failed intermission, but the window I chose there would do so. Are you thinking something like:

```
2245.2 "White Knight's Tour" sync /:Dawn Knight:1038:/
2254.5 "Sacred Flame Enrage?"

2294.0 "--sync--" /:Ser Charibert:1037:/ window 100,15
2300.0 "Altar Candle" sync /:Ser Charibert:1030:/ window 20,20
```

With no mention of 1037 in the timeline at all? That might be doable. In the event a group fails the intermission and somehow survives, it should pick up still.

Reply:
Does 1037 get used at time=2294 when the intermission fails, and earlier if it succeeds?